lib/truffle/rbconfig.rb

```diff
@@ -226,6 +233,7 @@ module RbConfig
     'sysconfdir'        => "#{prefix}/etc", # doesn't exist, as in MRI
     'target_cpu'        => host_cpu,
     'target_os'         => host_os,
+    'TEENY'             => teeny,
```

Comment:
Not sure whether MAJOR/MINOR/TEENY should be from RUBY_VERSION or RUBY_ENGINE_VERSION but since RUBY_ENGINE_VERSION is not always just 3 digits let's use RUBY_VERSION as you did here.